src/components/ListingDetails/index.tsx

```diff
@@ -0,0 +1,424 @@
+  // Languages
+  {
+    label: 'Language(s)',
+    getValue: data => data.languages.join(', '),
+  },
+  // Language Support Type
+  {
+    label: 'Language Support Type',
+    getValue: () => 'One-Time Intepretation',
+  },
+  // Remote/In Person
+  {
+    label: 'Remote/In Person',
+    getValue: data => (data.is_remote ? 'Remote' : 'In Person'),
+  },
+];
+
+const docFields: ListingField<DocumentTranslation>[] = [
+  // Languages
+  {
+    label: 'Language(s)',
+    getValue: data => data.languages.join(', '),
+  },
+  // Language Support Type
+  {
+    label: 'Language Support Type',
+    getValue: () => 'Document Translation',
+  },
+  // Number of Pages
+  {
+    label: 'Number of Pages',
+    getValue: data => data.num_pages.toString(),
+  },
+];
+
+const lcaFields: ListingField<LimitedCaseAssignment>[] = [
+  // Country Field
+  {
+    label: 'Country Field',
+    getValue: data => data.country,
+  },
+  // Language(s)
+  {
+    label: 'Language(s)',
+    getValue: data => data.languages.join(', '),
+  },
+  // Expected Deliverable
+  {
+    label: 'Expected Deliverable',
+    getValue: data => data.deliverable,
+  },
+];
+
+export default function ListingDetails({
+  listingData,
+  interpretation = false,
+}: {
+  listingData: Listing;
+  interpretation?: boolean;
+}) {
+  const auth = useAuth();
+  const profile = useProfile();  
+  
+  const listingFields = useMemo(() => {
+    let fields: ListingField<any>[];
+    if (listingData.listing_type === 'CASE') {
+      if (interpretation && listingData.needs_interpreter) {
+        fields = caseInterpretationFields;
+      } else if (listingData.is_detained) {
+        fields = [
+          ...caseFields,
+          {
+            label: 'Custody Location',
+            getValue: data => data.client_location || 'N/A',
+          },
+        ];
+      } else {
+        fields = [
+          ...caseFields,
+          {
+            label: 'Client Location',
+            getValue: data => data.client_location || 'N/A',
+          },
+        ];
+      }
+    } else if (listingData.listing_type === 'INT') {
+      fields = intepretationFields;
+    } else if (listingData.listing_type === 'DOC') {
+      fields = docFields;
+    } else {
+      fields = lcaFields;
+    }
+
+    return (
+      <InnerFieldContainer>
+        {fields.map(({ label, getValue }) => (
+          <div key={label}>{renderField(label, getValue(listingData))}</div>
+        ))}
+      </InnerFieldContainer>
+    );
+  }, [listingData, interpretation]);
+
+  const dateComponent = useMemo(() => {
+    if (listingData.listing_type !== 'INT') {
+      let dateHeader = 'Deadline';
+      let dateData = '';
+      if (listingData.listing_type === 'CASE') {
+        dateHeader = 'Next Court/Filing Date';
+        dateData = formatTimestamp(listingData.upcoming_date);
+      } else {
+        dateData = formatTimestamp(listingData.deadline);
+      }
+      return (
+        <IconTextGroup>
+          <Icon type="calendar" />
```

Comment:
I think this calendar icon is the one used in the `ListingCard` component so it looks a little smaller than expected. Could we add the calendar icon that's used in the `CaseDetails` on the Figma?